infos/anfi-faq.md

```diff
@@ -10,7 +10,7 @@ dass wir alles wissen bzw. dass wir auch immer alles korrekt wissen.
 Generell gilt daher immer, sich von offizieller Seite absichern, also in
 Vorlesungsverzeichnissen, Studienordnungen, auf Schwarzen Brettern, ...
 nachzuschauen und bei den
-[Studienberatern](http://www.wsi.uni-tuebingen.de/studium/ansprechpartner/studienberatung.html)
+[Studienberatern](https://uni-tuebingen.de/fr/74360)
```

Comment:
It seems like you fixed the links, but why do they refer to the french version?
The Website of the Fachbereich Informatik has no french version, so please correct it to .../de/...